doc/getting_started.md

```diff
@@ -221,30 +221,32 @@ everything is working correctly:
 ```console
 Creating user 'ot' with UID=1000, GID=1000.
 ot@ot-sca:/repo$ cd cw/cw305/
-ot@ot-sca:/repo/cw/cw305$ ./simple_capture_traces.py 
-Connecting and loading FPGA... Done!
+ot@ot-sca:/repo/cw/cw305$ ./capture.py --cfg-file capture_aes.yaml capture aes-random
+Connecting and loading FPGA... SKIPPED! (see: /home/abdullah/workspace/ot-sca/cw/cw305/util/device.py:initialize_fpga)
```

Comment:
Please revert this line as it assumes the FPGA has already been loaded previously. Otherwise it would just print:
```
Connecting and loading FPGA... Done!
```
as before.